tests/spec/test_enum.py

```diff
@@ -170,3 +169,37 @@ def test_has_thrift_module(loads):
         }
     ''')
     assert module is module.Foo.__thrift_module__
+
+
+def test_primitive_type_string(loads):
+    Answer = loads('''
+        enum Answer {
+            Yes, No, Unspecified
+        } (py.primitiveType = "string")
+    ''').Answer
+
+    spec = Answer.type_spec
+
+    assert spec.to_primitive(Answer.Yes) == 'Yes'
+    assert spec.to_primitive(Answer.No) == 'No'
+    assert spec.to_primitive(Answer.Unspecified) == 'Unspecified'
```

Comment:
To the above 3 lines, yeah, i could see how this feature would be useful. I wonder if we could get away from `py.primitiveType` and just go `primitiveType` where each language would implement something like this in it's presence.